app/mempool/senders_with_nonce.go

```diff
@@ -0,0 +1,63 @@
+package mempool
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authante "github.com/cosmos/cosmos-sdk/x/auth/ante"
+	"github.com/cosmos/cosmos-sdk/x/auth/signing"
+	evmtypes "github.com/evmos/ethermint/x/evm/types"
+)
+
+func GetSendersWithNonce(tx sdk.Tx) ([]SenderWithNonce, error) {
```

Comment:
Let's add comment to the function
We can describe why needed, we compute the signer differently for evm messages?